plugins/pulp_rpm/plugins/migrations/0011_new_importer.py

```diff
@@ -43,7 +42,7 @@ def _migrate_collection(type_id):
 
         # add these attributes, which we previously didn't track in the DB.
         package['size'] = int(primary_element.find('size').attrib['package'])
-        if type_id == RPM.TYPE:
+        if type_id == 'rpm':
```

Comment:
We should leave this alone for the same reason as [above](https://github.com/pulp/pulp_rpm/pull/741#discussion-diff-47707329).